### ISSUE TITLE
How to use voice files instead pure TTS?

### ISSUE THREAD

**Issue by Vadim2S** [neosapience/editts#1]
In papers you say about LJ speech dataset test (4.3 Content replacement). Can you provide code for loading voice files instead pure sample generation in tts.py?

**Comment by jaketae** [comment]
Hello @Vadim2S, thanks for opening this issue, and apologies for the belated reply. The modeling code is a direct adaptation of [Grad-TTS](https://github.com/huawei-noah/Speech-Backbones/tree/main/Grad-TTS#training), so you can refer to the upstream repository for detailed instructions on how to load data. Hope this helps!

**Comment by mvoodarla** [comment]
I have this question as well. Looking at the inference code, it is unclear how I could drop-in replace running Grad-TTS with my own source WAV file. Any tips would be appreciated :) Or if you have pointers to any other methods of doing something similar.

**Comment by jaketae** [comment]
Hey @mvoodarla, thanks for opening this issue. 

> how I could drop-in replace running Grad-TTS with my own source WAV file

Could you explain in more detail what you mean by this? I assume this is in the context of [content replacement](https://github.com/neosapience/editts/blob/master/edit_content.py).

**Comment by mvoodarla** [comment]
Yes, I would like to have the ability to submit a source wav file with or without a text transcription of it, and be able to replace a word or a set of words that was said in that source wav file. Does that make sense?
